frontend/src/components/App.js

```diff
@@ -1,36 +1,14 @@
 import React, { Component } from 'react';
```

Comment:
I don't know what is the difference between this file and the AppRoutes.js
why we have two apps?